### DISCUSSION TITLE
Generate reports with page views and custom events

### DISCUSSION THREAD

**Discussion by cbdeveloper** [plausible/analytics#2202] [Anything Else]
Hi everyone,

I'm looking into buying plausible and I would like to check if it would fit my use case.

I have a pages that display some products. On every visit on each page, I want to dispatch the page view, and also custom events for every click on each product of the page. The custom event should include information about the product.

Then I would like to generate the following reports.

For a specific page, I want to know how many clicks each product received.

For all pages, how many page views and how many product clicks each page received.

Is that doable? Will I need to export to excel? Or can I do it from the dashboard?

Is it possible to export the data to excel?

**Comment by metmarkosaric** [comment]
hi @cbdeveloper! this should be possible with our custom events and properties. see https://plausible.io/docs/custom-event-goals

you can view it in the dashboard and there's also the csv export: https://plausible.io/docs/export-stats

there's a free trial so feel free to check it out and see whether Plausible is a good solution for you. have fun!
